src/plugins/plugin.ts

```diff
@@ -23,12 +23,14 @@ export const PluginUI: ReadonlySet<PluginConfig> = plugins
 
 import { GitcoinPluginDefine } from './Gitcoin/define'
 import { RedPacketPluginDefine } from './RedPacket/define'
+import { PollsPluginDefine } from './Polls/define'
 import { StorybookPluginDefine } from './Storybook/define'
 import { FileServicePluginDefine } from './FileService/define'
 import { TraderPluginDefine } from './Trader/define'
 import { Flags } from '../utils/flags'
 plugins.add(GitcoinPluginDefine)
 plugins.add(RedPacketPluginDefine)
+plugins.add(PollsPluginDefine)
```

Comment:
Does it work for mobile? Maybe it need a new flag like `if (Flags.poll_enabled) plugins.add(PollsPluginDefine)`

Reply:
I see you marked it as resolved but I don't see the code change